expander.go

```diff
@@ -1078,7 +1083,7 @@ func expandResponse(response *Response, resolver *schemaLoader, basePath string)
 
 	parentRefs = parentRefs[0:]
```

Comment:
I never quite understood this line. As far as I remember gometalinter / golangci-lint detect is a useless. Thought?

Reply:
I think it is just a save guard to reset the parent refs array before we expand a schema. I don't know if it is needed, but it is better to be safe I guess.